docs/data_types/audio/audio.md

```diff
@@ -0,0 +1,199 @@
+# 🔊 Audio
+
+DocArray supports many different modalities including `Audio`.
+This section will show you how to load and handle audio data using DocArray.
+
+Moreover, we will introduce DocArray's audio specific types, to represent your audio data ranging from [`AudioUrl`][docarray.typing.url.AudioUrl] to [`AudioBytes`][docarray.typing.bytes.AudioBytes] and [`AudioNdArray`][docarray.typing.tensor.audio.audio_ndarray.AudioNdArray].
+
+!!! note
+    This requires a `pydub` dependency. You can install all necessary dependencies via:
+    ```cmd 
+    pip install "docarray[audio]"
+    ```
+
+## Load audio file
+
+First, let's define a class, which extends [`BaseDoc`][docarray.base_doc.doc.BaseDoc] and has an `url` attribute of type [`AudioUrl`][docarray.typing.url.AudioUrl], and an optional `tensor` attribute of type `AudioTensor`.
+
+Next, we can instantiate an object of that class with a local or remote url. 
+
+```python
+from docarray import BaseDoc
+from docarray.typing import AudioUrl, AudioNdArray
+
+
+class MyAudio(BaseDoc):
+    url: AudioUrl
+    tensor: AudioNdArray = None
+    frame_rate: int = None
+
+
+doc = MyAudio(
+    url='https://github.com/docarray/docarray/blob/feat-rewrite-v2/tests/toydata/hello.mp3?raw=true'
+)
+```
+
+Loading the content of the audio file is as easy as calling [`.load()`][docarray.typing.url.AudioUrl] on the [`AudioUrl`][docarray.typing.url.AudioUrl] instance. 
+
+This will return a tuple of:
+
+- an [`AudioNdArray`][docarray.typing.tensor.audio.AudioNdArray] representing the audio file content 
+- the corresponding frame rate describing the number of signals for a certain period of time
+
+```python
+doc.tensor, doc.frame_rate = doc.url.load()
+doc.summary()
+```
+<details>
+    <summary>Output</summary>
+    ``` { .text .no-copy }
+    📄 MyAudio : 2015696 ...
+    ╭──────────────────────┬───────────────────────────────────────────────────────╮
+    │ Attribute            │ Value                                                 │
+    ├──────────────────────┼───────────────────────────────────────────────────────┤
+    │ url: AudioUrl        │ https://github.com/docarray/docarray/blob/feat-rew    │
+    │                      │ ... (length: 90)                                      │
+    │ tensor: AudioNdArray │ AudioNdArray of shape (30833,), dtype: float64        │
+    │ frame_rate: int      │ 44100                                                 │
+    ╰──────────────────────┴───────────────────────────────────────────────────────╯
+    ```
+</details>
+
+
+## AudioTensor
+
+DocArray offers several AudioTensors to store your data to:
+
+- [`AudioNdArray`][docarray.typing.tensor.audio.audio_ndarray.AudioNdArray]
+- `AudioTorchTensor`
+- `AudioTensorFlowTensor`
```

Comment:
Can we also link to the torch and tf variants? why does only the np one have the link?